common/src/Utilities/Console.cpp

```diff
@@ -453,6 +453,28 @@ bool IConsoleWriter::Warning(const wxString fmt, ...) const
     return false;
 }
 
+bool IConsoleWriter::WriteLn(ConsoleColors color, const std::string& str) const
```

Comment:
Am I reading this right that with `fromUTF8` using a string_view, **all** those methods could use it too?